setup.cfg

```diff
@@ -17,21 +17,25 @@ classifiers =
     Development Status :: 3 - Alpha
     Intended Audience :: Developers
     License :: OSI Approved :: MIT License
-    Programming Language :: Python :: 3
-    Programming Language :: Python :: 3.8
+    Operating System :: OS Independent
+    Programming Language :: Python :: 3 :: Only
     Programming Language :: Python :: 3.9
     Programming Language :: Python :: Implementation :: CPython
     Programming Language :: Python :: Implementation :: PyPy
+    Topic :: Software Development
+    Topic :: Utilities
 platforms = any
 
 # setuptools options
 [options]
 packages = compass
 include_package_data = True
-python_requires = >=3.8
+python_requires = >=3.9
```

Comment:
Think this should be 3.8? (ref requirements.txt?)

Reply:
I moved to generic  types (dict instead of typing.Dict) which requires 3.9.

I didn't think you could specifiy a python version in `requirements.txt`? I'm a reasonably recent convert to conda for depencency management, it takes some adjustment but is pretty nice

Reply:
Have done a search for `3.8` and `38` across all files and couldn't find anything apart from two yaml files, which is odd

Reply:
Ignore me - I was referencing the wrong file, pyproject.toml was where I read 3.8 and the rollback from 3.9 comment due to lack of black support.

Reply:
And that line I blocked mentally as I knew it was _meant_ to be 3.9! Crossed wires & all that!! 

Will push those couple changes & merge & release